dir.props

```diff
@@ -156,35 +158,29 @@
     <RunApiCompat>true</RunApiCompat>
   </PropertyGroup>
 
-  <!--
-  Projects that have no OS-specific implementations just use Debug and Release for $(Configuration).
-  Projects that do have OS-specific implementations use OS_Debug and OS_Release, for all OS's we support even
-  if the code is the same between some OS's (so if you have some project that just calls POSIX APIs, we still have
-  OSX_[Debug|Release] and Linux_[Debug|Release] configurations.  We do this so that we place all the output under
-  a single binary folder and can have a similar experience between the command line and Visual Studio.
-  -->
 
-  <!--
-  If Configuration is empty that means we are not being built in VS and so folks need to explicitly pass the different
-  values for $(ConfigurationGroup), $(TargetGroup), or $(OSGroup) or accept the defaults for them.
-  -->
   <PropertyGroup>
-    <ConfigurationGroup Condition="'$(ConfigurationGroup)'==''">Debug</ConfigurationGroup>
-<!-->    <TargetGroup Condition="'$(TargetGroup)' == ''">netcoreapp1.1</TargetGroup>
-    <OSGroup Condition="'$(OSGroup)' == ''">$(OSEnvironment)</OSGroup>
--->
-    <Configuration Condition="'$(OSGroup)' != 'AnyOS'">$(TargetGroup)-$(OSGroup)</Configuration>
-    <Configuration Condition="'$(OSGroup)' == 'AnyOS'">$(TargetGroup)</Configuration>
+    <BuildConfigurationImportFile>$(BinDir)tools/configuration/configuration.props</BuildConfigurationImportFile>
   </PropertyGroup>
 
-  <!-- Set up Default symbol and optimization for Configuration -->
-  <Choose>
-    <When Condition="'$(ConfigurationGroup)'=='Debug'">
-      <PropertyGroup>
-        <DebugSymbols Condition="'$(DebugSymbols)' == ''">true</DebugSymbols>
-        <Optimize Condition="'$(Optimize)' == ''">false</Optimize>
-        <DebugType Condition="'$(DebugType)' == ''">full</DebugType>
-        <DefineConstants>$(DefineConstants),DEBUG,TRACE</DefineConstants>
+  <Import Project="$(BuildConfigurationImportFile)" Condition="Exists('$(BuildConfigurationImportFile)') and '$(BuildConfigurations)' != ''" /> 
+
+  <!-- Set up Default symbol and optimization for Configuration --> 
+  <Choose> 
```

Comment:
Can you please file and issue to move these properties and the Targets* properties to the generated props files.